### ISSUE TITLE
Change machine names in the machine's settings

### ISSUE THREAD

**Issue by ETA-2024** [FoldingAtHome/fah-web-client-bastet#142]
It makes more sense to me to edit a machine's name in that machines settings, rather than in the account settings, because this is how it worked in FAHControl.

Please implement this functionality.

**Comment by muziqaz** [comment]
I'm not at my PC right now, but when I check my farm remotely through the phone, every single system name can be changed from each machine's settings section

**Comment by jcoffland** [comment]
Right, this is how it already works.  If there's a bug provide screenshots.  Also, make sure you're on the latest version.

**Comment by Schrodingers-cat-too** [comment]
Thanks @jcoffland and @muziqaz.  

I'm not using version 8 now, but I believe this post was hidden when I first posted it, so I reopened it in case it was still relevant.

It may have been version 8.3.5 that I was using at that time, so I'm not surprised (and pleased) that machine names can now be edited in the machine's settings page.
